content/participants/ttjc@killcodeX/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Aaquib Ahmed
+type: "participant"
+name: "Aaquib Ahmed"
+portfolio: "http://aaquib.netlify.app/"
+github-link: "https://github.com/killcodeX"
+linkedin-link: "https://linkedin.com/in/aaquib-ahmed/"
+looking-for: "job"
+description: "FrontEnd Developer | Brushing React"
+---
+
+Hi! I am Aaquib Ahmed , pursuing my b-tech from Bangalore Institute of Technology. I am also a freelancer blogger, graphics designer and web developer. [link](http://aaquib.netlify.app/)
+
+### introduction
+
+I am currently working as FrontEnd Developer in Skimbox [link](http://skimbox.co/), from my freshman year I started working as part time blogger and content writer through which I also oppurtunity to work in different domains like SEO and Social Media Marketing, but I always had love for coding, I used to code simultaneously on HACKERANK [link](https://www.hackerrank.com/diazzaaquib), then finaly in my third year I started working on small web projects and started doing my Frelancing job.
```

Comment:
Hey! @killcodeX a small mistake here:
* Frelancing
* Also just a suggestion instead of using the word **link** everywhere you can do something like:
  ```
  [Skimbox](http://skimbox.co/)
  ```